app/screens/route-details/route-details-screen.tsx

```diff
@@ -49,6 +57,21 @@ export const RouteDetailsScreen = observer(function RouteDetailsScreen({ route }
   const openFirstRideAlertSheet = () => {
     bottomSheetRef.current?.expand()
   }
+  const eventConfig: CreateOptions = {
+    title: translate("plan.trainFromStation") + ' '
+      + route.params.routeItem.trains[0].originStationName+ ' '
+      + translate("plan.toStation")+ ' '
+      + route.params.routeItem.trains[0].destinationStationName
+    ,
```

Comment:
Use existing texts from i18n files:

Train No. 422 From Tel Aviv - HaShalom To Karmiel.